.github/workflows/test.yml

```diff
@@ -24,13 +27,17 @@ jobs:
       fail-fast: false
       matrix:
         k3s-channel:
-        - v1.19
-        helm-version:
-        - v3.5.0
+        # Available channels: https://github.com/k3s-io/k3s/blob/HEAD/channel.yaml
+        - latest
         test:
         - main
         - auth
         - helm
+        include:
+          # a job specific for the oldest supported k8s / helm cli version
+          - k3s-channel: "1.17"
```

Comment:
```suggestion
          - k3s-channel: "v1.17"
```

Reply:
I think both may work, but this is how they are referenced in the channel list and how we previously referenced them.